### ISSUE TITLE
Pokestop out of range

### ISSUE THREAD

**Issue by epicblood** [OpenPoGo/OpenPoGoBot#138]
### Expected Behavior

After arriving at a pokestop, it should spin once in range.
### Actual Behavior

Arrives at pokestop, spins, then logs it is out of range and moves on.

```
[2016-07-31 13:03:05] [#] Walking Finished
[2016-07-31 13:03:05] [#] Arrived at Pokestop
[2016-07-31 13:03:08] [#] Now at Pokestop: Sheep Meadow
[2016-07-31 13:03:08] Spinning...
[2016-07-31 13:03:11] [#] Pokestop out of range
[2016-07-31 13:03:12] [#] Found fort 29c95671e7af49538ceb15a20143cc92.16 at distance 0.12km
[2016-07-31 13:03:12] [#] Need to move closer to Pokestop
[2016-07-31 13:03:12] [#] Asking google for directions
```
### Steps to Reproduce

Doesn't always happen, let it run for awhile and it will happen now and then. Tested in central park.
Noticed this happening after the 'ask google for directions' PR

**Comment by andythorne** [comment]
I think I know what may be causing this.

Basically when we ask google for directions in may snap the destination coords to the nearest road/path/address/free space. This should be fixed with #129.

**Comment by andythorne** [comment]
Ok after further investigation it I've found out why. 
- Google Directions could sometimes be slightly off, which will be fixed in #136 .
- When walking, the bot works out how many step we need to take to cover the distance, given the walk speed (default 4.16). Most times the steps require is not a round number (e.g. we may have to take 10.8 steps). In this case the number is rounded (to 11) and we take 11 steps plus a small random delta to make each step different. So, sometimes we can overshoot or undershoot the destination by a small amount. I don't really see this as a problem as it would look quite suspicious if we kept visiting the latitude and longitude location of a pokestop.

I think this issue was a combination of them both and should be resolved after #136
